ui/components/multichain/badge-status/badge-status.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useSelector } from 'react-redux';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderColor,
+  BorderRadius,
+  Display,
+  JustifyContent,
+  Size,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarAccount,
+  AvatarAccountVariant,
+  BadgeWrapper,
+  Box,
+} from '../../component-library';
+import {
+  getUseBlockie,
+} from '../../../selectors';
+import Tooltip from '../../ui/tooltip';
+
+export const BadgeStatus = ({
+  className,
+  badgeBackgroundColor,
+  badgeBorderColor,
```

Comment:
Should the default be the not-connected color?

Reply:
Updated!